decidim-budgets/spec/helpers/decidim/budgets/projects_helper_spec.rb

```diff
@@ -16,6 +16,13 @@ module Budgets
       let(:address) { "Carrer Pic de Peguera 15, 17003 Girona" }
       let(:latitude) { 40.1234 }
       let(:longitude) { 2.1234 }
+      let(:redesign_enabled) { false }
+
+      before do
+        # rubocop:disable RSpec/AnyInstance
+        allow_any_instance_of(ActionView::Base).to receive(:redesign_enabled?).and_return(redesign_enabled)
```

Comment:
Could we move this to a shared context that is included in the target spec as this seems to be repeated across multiple specs?